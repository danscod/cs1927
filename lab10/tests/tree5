5 1 9 4 8 3 2 7 6
