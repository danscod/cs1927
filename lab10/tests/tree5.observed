Tree:
    5
   / \
  3   7
 /     \
1       9

#nodes = 5,  #leaves = 2
Infix  : 1 3 5 7 9 
Prefix : 5 3 1 7 9 
Postfix: 1 3 9 7 5 
ByLevel: 5 3 7 1 9 
