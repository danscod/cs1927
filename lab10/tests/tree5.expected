Tree:
      5
     / \
    /   \
   /     \
  1       9
   \     /
    4   8
   /   /
  3   7
 /   /
2   6

#nodes = 9,  #leaves = 2
Infix  : 1 2 3 4 5 6 7 8 9 
Prefix : 5 1 4 3 2 9 8 7 6 
Postfix: 2 3 4 1 6 7 8 9 5 
ByLevel: 5 1 9 4 8 3 7 2 6 
